lib/events/roomcreateevent.cpp

```diff
@@ -3,8 +3,30 @@
 
 #include "roomcreateevent.h"
 
+#include <array>
+
 using namespace Quotient;
 
+constexpr inline auto roomTypeStrings = std::to_array({
```

Comment:
```suggestion
constexpr inline auto roomTypeStrings = to_array({
```
`std::to_array` is too new and doesn't exist in some of the used standard libraries.